rasa_sdk/forms.py

```diff
@@ -40,6 +40,8 @@ def from_entity(
         entity: Text,
         intent: Optional[Union[Text, List[Text]]] = None,
         not_intent: Optional[Union[Text, List[Text]]] = None,
+        role: Optional[Text] = None,
+        group: Optional[Text] = None,
```

Comment:
Should these also optionally be lists? So that you could provide e.g. 2 out of 3 roles it could be?

Reply:
That should not happen. You also need to decide for just ONE entity label. Same should happen for role and group.

Reply:
That's true, I guess you could provide 2 mappings if for some reason you wanted to be able to pick up from both

Reply:
You mean in the slot mapping itself, right? So nothing I need to update.

Reply:
nope, you should be able to provide a list i think

Reply:
e.g.
```
return {
                "some_slot": [
                      self.from_entity(entity="some_entity", role="role1"),
                      self.from_entity(entity="some_entity", role="role2")
                ]
            }
```
this is how you do it if you wanted to have it pick up from one of 2 different entities

Reply:
ah ok, great.